### ISSUE TITLE
Messages sent to invalid mail addresses are marked as "Delivered" in "Message details"

### ISSUE THREAD

**Issue by gerryfrancis** [deltachat/deltachat-desktop#1628]
- Platform (linux/mac/windows):
Windows 8.1 Pro.

- Delta Chat Desktop Version:
1.3.2.

- Expected behavior:
When starting a chat with a new member and entering an invalid (non-existant) mail address (e.g. due to a typo), the window "Message details" should show status "Sent" instead of "Delivered" (green tick).

- Actual behavior:
When starting a chat with a new member and entering an invalid (non-existant) mail address (e.g. due to a typo), the window "Message details" show status "Delivered" (green tick).

- Steps to reproduce the problem:
-- Select "New chat" from the three-dot menu.
-- Enter an invalid mail address.
-- Write a message and send it.

- Remark:
Actually we cannot check if the message has been delivered unless we read the "Undelivered Mail" reports that are received from the mail provider.

**Comment by Jikstra** [comment]
This is not really easy to implement, because we actually send this message. No  matter if the address is valid or not, if we send it out via SMTP it's getting sent and we therefore also think this was a valid sent message. But somehow parsing those Undelivered Mail reports would be quite neat, but of course a core feature and not something we can do on the desktop side of things.

**Comment by gerryfrancis** [comment]
@Jikstra I agree, this would not be an easy task to solve. Nevertheless, the term "Delivered" only applies for messages that really arrived on the other side. I guess it would be better if "Delivered" is changed to "Sent", otherwise a user of Delta Chat would think that the message sent was indeed delivered.

**Comment by Jikstra** [comment]
Yup I agree, sent makes more sense. The only way that we know the other side received it, is if we get a read notification.

**Comment by gerryfrancis** [comment]
> The only way that we know the other side received it, is if we get a read notification.

@Jikstra As matters stand, you are right.
